src/StoreContext.jsx

```diff
@@ -47,9 +54,4 @@ StoreProvider.defaultProps = {
   initialValues: null
 };
 
-StoreProvider.childContextTypes = {
-  api: PropTypes.func,
-  store: PropTypes.object
-};
-
-export default StoreProvider;
+export const StoreConsumer = StoreContext.Consumer;
```

Comment:
Super minor but maybe move this export to below `export const StoreContext = createContext({});`? Confused me at first when I looked at this file.

Reply:
Yeah i was just following the most common pattern i could fine but i agree